src/plugins/moreUserTags.tsx

```diff
@@ -81,51 +98,89 @@ const tags: Tag[] = [
         permissions: ["MOVE_MEMBERS", "MUTE_MEMBERS", "DEAFEN_MEMBERS"]
     }
 ];
+const defaultSettings = Object.fromEntries(
+    tags.map(({ name, displayName }) => [name, { text: displayName, showInChat: true, showInNotChat: true }])
+) as TagSettings;
+
+function SettingsComponent(props) {
+    if (!settings.store.tagSettings) settings.store.tagSettings = defaultSettings;
+    const [tagSettings, setTagSettings] = useState(settings.store.tagSettings ?? defaultSettings as TagSettings);
```

Comment:
shouldn't need this due to above line

```suggestion
    const [tagSettings, setTagSettings] = useState(settings.store.tagSettings as TagSettings);
```